content/docs/guides/javascript/js-api.md

```diff
@@ -116,6 +116,19 @@ The following methods and configuration options accept a function taking a `Cont
 - `preOps()`
 - `postOps()`
 
+## Referencing includes files
+
+Just like in `.sqlx` files, you can reference any `includes` function, macro or constant inside a `.js` file. However, the syntax differs slightly.
+
+For example, if you would like to reference `SERIVCE_NAME` and `SERVICE_ID` from the `includes/service` file, you must declare these constants using the JavaScript builtin `require` function.
```

Comment:
nit: mispelling of `SERVICE_NAME`, also would suggest you refer to `includes/service.js` just to make it clear it's a JS file you're referring to

Reply:
Well, that's embarrassing. Fixed!